### PR TITLE
4-15-23 Spring Ball

### PR DESCRIPTION
**Pull request by MonaraMir**
## About The Pull Request
Event map for the ball on 4/15/23
**Do not merge** unless contacted.
Map will likely stay active as a replacement to the current Class M Gaia, but will be unmerged *or* have a patch for spawn point changes before/after the event.

## Why It's Good For The Game

Event Map + lots of fixes. 

## Changelog
Added gokarts
Rebuilt Gaia
Added Concrete floors
Fixed desert sand (only one subtype)

:cl:
add: Added gokarts
add: Rebuilt Gaia
add: Added Concrete floors
fix: Fixed desert sand (only one subtype)
/:cl:

### CODE REVIEW

File: code/modules/maps/turf_makers/planetary_turfs.dm
Comment (TheLordME) [inline code/modules/maps/turf_makers/planetary_turfs.dm:1]: we should probably write an outdoors gaia turf macro, but thats nothing I expect off you

File: code/modules/maps/weather/classm.dm [hunk 1]
```diff
@@ -11,7 +11,7 @@
 
 
 /datum/time/classm
-	seconds_in_day = 3 HOURS
+	seconds_in_day = 10 HOURS
```
Comment (TheLordME) [inline code/modules/maps/weather/classm.dm:14]: understandable for the ball, but should this be a permanent change?

File: code/modules/power/lighting/lights.dm [hunk 2]
```diff
@@ -111,6 +111,20 @@
 	nightshift_range = 4
 	nightshift_power = 0.4
 
+/obj/item/light/bulb/strong
```
Comment (TheLordME) [inline code/modules/power/lighting/lights.dm:114]: is this supposed to be unobtainable? if no, please add them to the lathe (https://github.com/Citadel-Station-13/Citadel-Station-13-RP/pull/5283 for what to do)

**Review by TheLordME — CHANGES_REQUESTED** [review]

File: code/game/objects/structures/curtains.dm
[the remark this answers is not in the document]
Comment (MonaraMir) [reply code/game/objects/structures/curtains.dm:1]: Needed a subtype for open varients

File: code/modules/maps/turf_makers/planetary_turfs.dm
Comment (MonaraMir) [reply code/modules/maps/turf_makers/planetary_turfs.dm:1]: Yeah it sucks

File: code/modules/maps/weather/classm.dm (hunk 1 above)
Comment (MonaraMir) [reply code/modules/maps/weather/classm.dm:14]: Yes, day cycles were stuck to permanently nighttime at 3-5 hours

File: code/modules/power/lighting/lights.dm (hunk 2 above)
Comment (MonaraMir) [reply code/modules/power/lighting/lights.dm:114]: Yes, these are only used in one type of street lamp that has a small bulb sprited, but needs the 8 tile radius of a strong bulb/light tube

**Review by TheLordME — APPROVED** [review]

**Pull request merged**